curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-plus-operator.md

```diff
@@ -44,10 +44,10 @@ You should use the `+` operator to build `myStr`.
 assert(code.match(/(["']).*\1\s*\+\s*(["']).*\2/g));
 ```
 
-`myStr` should be created using the `var` keyword.
+`myStr` should be created using the `const` keyword.
 
 ```js
-assert(/var\s+myStr/.test(code));
+assert(/const\s+myStr/.test(code));
 ```
 
 You should assign the result to the `myStr` variable.
```

Comment:
Thanks for catching this @ShaunSHamilton.

I played around with this for a bit, and it seems like the easiest thing to do is to change the seed to `let myStr;`, similar to the original version of the challenge.

Briefly considered `const myStr;`, but that would lead to an error when the page loads, and I'm not sure how the automated tests would handle that.